content/spin/cli-reference.md

```diff
@@ -3361,7 +3347,24 @@ CLI commands have four phases that indicate levels of stability:
 
 {{ blockEnd }}
 
-{{ startTab "v1.1.0"}}
+{{ startTab "v1.1"}}
+
+| Command                                                                               | Stability    |
+| ------------------------------------------------------------------------------------- | ------------ |
+| <code>spin add</code>                                                                 | Stable       |
+| <code>spin build</code>                                                               | Stable       |
+| <code>spin new</code>                                                                 | Stable       |
+| <code>spin plugins <install&vert;list&vert;uninstall&vert;update&vert;upgrade></code> | Stable       |
+| <code>spin templates <install&vert;list&vert;uninstall&vert;upgrade></code>           | Stable       |
+| <code>spin up</code>                                                                  | Stable       |
+| <code>spin cloud <deploy&vert;login></code>                                           | Stabilizing  |
+| <code>spin registry</code>                                                            | Stabilizing  |
+| <code>spin watch</code>                                                               | Experimental |
```

Comment:
This was given as `stabilising` before - was that an error?  It can't have retrospectively gone backwards can it?!

Reply:
(Which raises the question of what does it take to count it as "stabilising" - it's given as "experimental" in 1.3.)

Reply:
It was stabilizing in 1.1, and moved to experimental in 1.2, so I'd assume the 1.1 was an error to begin with. Marking this experimental in this PR. We can fix in a subsequent if this is wrong.